.github/workflows/comment.yml

```diff
@@ -0,0 +1,62 @@
+name: Comment on clippy
+
+on:
+  workflow_run:
+    workflows: ["Rust"]
+    types:
+      - completed
+
+jobs:
+  upload:
```

Comment:
Why is this named "upload"?  Isn't it doing a download?

Reply:
From the example, but I can name it whatever.